package.json

```diff
@@ -33,7 +33,7 @@
     "polymer-analyzer": "2.0.0-alpha.40",
     "polymer-bundler": "2.0.0-pre.14",
     "polymer-project-config": "^2.0.0",
-    "sw-precache": "^4.2.0",
+    "sw-precache": "^5.0.0",
```

Comment:
5.1.1 is the latest version:
https://github.com/GoogleChrome/sw-precache/releases/tag/5.1.1